### ISSUE TITLE
Add peas-ng tome

### ISSUE THREAD

**Issue by hulto** [KCarretto/realm#401]
**Is your feature request related to a problem? Please describe.**
Would be nice to have an easy way to run peas-ng to do host recon during CTFs.

**Describe the solution you'd like**
Create a tome with a bundled file from the peas-ng release.
Download the peas-ng binary through the assets module.
Copy it to disk into a temp file.
Execute it saving the output to a temp file.
Once finished copy back the output and delete both the peas-ng binary and the output file.

**Describe alternatives you've considered**
Na

**Additional context**
- Currently blocked by the realm file download API integration with imix.
